### DISCUSSION TITLE
3.2.6

### DISCUSSION THREAD

**Discussion by michaelwillis** [michaelwillis/dragonfly-reverb#127] [General]
* Attempt at fixing bug on HiDPI displays
* Universal MacOS installer
* ARM builds for Linux

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelwillis/dragonfly-reverb/releases/tag/3.2.6'>3.2.6</a>.</em>

**Comment by cpsmusic** [comment]
Just tried out the new version - the window size problem is fixed but the overall size of the GUI is too small. It's tiny!

Here's an image of Dragonfly Plate next to bx_console E at 100%.

<img width="1748" alt="Screen Shot 2022-03-31 at 3 18 23 pm" src="https://user-images.githubusercontent.com/6424857/160978589-7f826e1b-9aa5-46f3-b101-65fffd6a9b91.png">

Cheers,

Chris

**Comment by DarceyLloyd** [comment]
Bitdefender - Blocked the download
dragonfly-reverb-win64-v3.2.6.zip tried to load a malicious resource detected as Trojan.Ceram.Gen.1 and was blocked.

Previous version downloads fine (3.2.5)
